app/todos/completed-todo.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import {
+  View,
+  Text,
+  FlatList,
+  StyleSheet,
+  ImageBackground,
+  Dimensions,
+} from "react-native";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { SafeAreaView } from "react-native-safe-area-context";
+
+type Todo = {
+  id: string;
+  text: string;
+  completed: boolean;
+};
+
+const CompletedTodos = () => {
+  const [completedTodos, setCompletedTodos] = useState<Todo[]>([]);
+
+  useEffect(() => {
+    loadCompletedTodos();
+  }, []);
+
+  const loadCompletedTodos = async () => {
+    const storedTodos = await AsyncStorage.getItem("todos");
+    if (storedTodos) {
+      const todos = JSON.parse(storedTodos);
+      setCompletedTodos(todos.filter((todo: Todo) => todo.completed));
+    }
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ImageBackground
+        source={require("../../assets/images/bg-2.jpeg")}
+        style={styles.image}
+        resizeMode="cover"
+      />
+      <Text style={styles.header}>Completed Todos</Text>
+      <FlatList
+        showsVerticalScrollIndicator={false}
+        contentContainerStyle={styles.todos}
+        data={completedTodos}
+        keyExtractor={(item) => item.id}
+        renderItem={({ item }) => (
+          <View style={styles.listItem}>
+            <Text style={styles.text}>{item.text}</Text>
+          </View>
+        )}
+      />
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
I think you should also try using nativewind. It'll make your work easier

Reply:
i'm not really familiar with native wind, but ill try it out. thank you

Reply:
It's similar to tailwind css, it's just the installation process that is slightly different